Repository/JobStatusRepository.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace Markup\JobQueueBundle\Repository;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\Query;
+use Markup\JobQueueBundle\Entity\JobStatus;
+
+class JobStatusRepository
+{
+    /** @var ManagerRegistry */
+    private $doctrine;
+
+    public function __construct(ManagerRegistry $doctrine)
+    {
+        $this->doctrine = $doctrine;
+    }
+
+    public function isUserEnabled(string $command, array $arguments): bool
+    {
+        /** @var JobStatus $jobStatus */
+        $jobStatus = $this->getEntityRepository()->findOneBy([
+            'command' => $command,
+            'arguments' => ($arguments) ? json_encode($arguments) : null
+        ]);
+
+        if ($jobStatus instanceof JobStatus) {
+            return $jobStatus->getEnabled();
+        }
+
+        return true;
+    }
+
+    public function findBy(array $arguments): ?JobStatus
+    {
+        $jobStatus = $this->getEntityRepository()->findOneBy($arguments);
+
+        if ($jobStatus instanceof JobStatus) {
+            return $jobStatus;
+        }
+
+        return null;
+    }
+
+    public function store(JobStatus $jobStatus): void
```

Comment:
please use save as a name, that will keep it consistent across the entire codebase